.github/workflows/e2e.yml

```diff
@@ -95,17 +96,9 @@ jobs:
         CI: "true"
       run: |
         source .venv/bin/activate
-        sudo apt-get update
-        sudo apt-get install -y libmagic-dev poppler-utils libreoffice
-        make install-pandoc
-        sudo add-apt-repository -y ppa:alex-p/tesseract-ocr5
-        sudo apt-get update
-        sudo apt-get install -y tesseract-ocr
-        sudo apt-get install -y tesseract-ocr-kor
-        sudo apt-get install diffstat
-        tesseract --version
         sudo make install-docker-compose
         docker compose version
+        pip freeze
```

Comment:
Is this just for development?

Reply:
This is intentional to know what the final version of dependencies are being used in case things get installed in a particular order when setting up the environment.